### ISSUE TITLE
Add absolutely trivial template

### ISSUE THREAD

**Issue by rpglover64** [commercialhaskell/stack-templates#39]
I want a template that _just_ creates a `stack.yaml` file (mostly so that I don't have to manually select the resolver). IIUC, this is just a blank `.hsfiles` with a particular name, e.g. `yaml-only.hsfiles`.

**Comment by bitemyapp** [comment]
`stack init`?

**Comment by rpglover64** [comment]
No cabal file.

**Comment by mgsloan** [comment]
Hmm, I'm not sure if I see the point of generating a stack.yaml file without having a cabal file.  In other words, if you're going to write your own cabal file, why not do that first and then stack init?

The main use I can think of for a stack.yaml without a cabal file is to provide a context for running ghci / single file scripts.  For that special case, it seems fine to manually look up available stackage snapshots.

So, in summary, I'm not opposed to this idea, but I haven't been convinced that it's useful.

**Comment by rpglover64** [comment]
I could also run multiple-file scripts, or play around with Liquid Haskell in its own sandbox, or play around with ghc-js, or with ghc-8 rc. It's useful if you're treating `stack` as a way to get better sandboxing rather than for project management.

**Comment by rpglover64** [comment]
This issue can be closed after commercialhaskell/stack/issues/2231 is, since I will be able to add the template myself.

**Comment by sboosali** [comment]
try:

    stack new lts6 https://raw.githubusercontent.com/sboosali/config/master/stack/templates/raw.hsfiles

it only has a stack.yaml and trivial {{name}}.cabal

You can then edit the compiler/resolver/build-depends

**Comment by snoyberg** [comment]
Issue #2231 is implemented, closing.
